Documentation/ceph-cluster-crd.md

```diff
@@ -856,9 +868,12 @@ spec:
   external:
     enable: true
   dataDirHostPath: /var/lib/rook
```

Comment:
- Is the dataDirHostPath used by anything if they don't have the admin key to start additional daemons or can we comment it out as well?
- To keep the example simple for the case of minimal keys, I wonder if we should have a separate example for the properties you can use with the admin key. It helps the examples be more readable if properties aren't commented out.